frontend/src/pages/team/Devices/dialogs/DeviceCredentialsDialog.vue

```diff
@@ -27,6 +27,7 @@
                 <ff-button kind="danger" class="ml-4" @click="regenerateCredentials()">Regenerate credentials</ff-button>
             </template>
             <template v-else>
+                <ff-button v-if="!!clipboardSupported" kind="secondary" @click="copyToClipboard()">Copy to Clipboard</ff-button>
```

Comment:
The !! Is redundant here I think clipboardSupported is always a Boolean

Reply:
DOH! Yes. Originally I had `!! navigation.clipboard` but the navigation object wasn't ready so had to wrap it in a data value and mounted method.  

```suggestion
                <ff-button v-if="clipboardSupported" kind="secondary" @click="copyToClipboard()">Copy to Clipboard</ff-button>
```